content/en/blog/_posts/2022-08-15-local-storage-capacity-isolation-ga.md

```diff
@@ -0,0 +1,94 @@
+---
+layout: blog
+title: "Kubernetes 1.25: Local Storage Capacity Isolation Reaches GA"
+date: 2022-08-15
+slug: local-storage-capacity-isolation-ga
+---
+
+**Author:** Jing Xu (Google)
+
+Local ephemeral storage capacity isolation was introduced as a alpha feature in Kubernetes 1.7 and it went beta in 1.9 and with Kubernetes 1.25 we are excited to announce general availability(GA) of this feature.
+
+Pods use ephemeral local storage for scratch space, caching, and for logs. The lifetime of local ephemeral storage does not extend beyond the life of the individual pod. It is exposed to pods using the container’s writable layer, logs directory, and EmptyDir volumes. Before this feature available, there have been issues related to the lack of local storage accounting and isolation, such as Pods do not know how much local storage is available and cannot request guaranteed local storage. Local storage is a best effort resource and pods can be evicted due to other pods filling the local storage.
```

Comment:
I'm not sure if EmptyDir needs to be encapsulated in a codeblock. I did it. 
```suggestion
Pods use ephemeral local storage for scratch space, caching, and logs. The lifetime of local ephemeral storage does not extend beyond the life of the individual pod. It is exposed to pods using the container’s writable layer, logs directory, and `EmptyDir` volumes. Before this feature was introduced, there were issues related to the lack of local storage accounting and isolation, such as Pods not knowing how much local storage is available and being unable to request guaranteed local storage. Local storage is a best-effort resource and pods can be evicted due to other pods filling the local storage.
```

Reply:
updated